### ISSUE TITLE
Popup's close button has too low contrast

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#348]
The popup's close button fails all of the following <span title="Web Content Accessibility Guidelines">WCAG</span> success criteria:
- [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
- [1.4.6 Contrast (Enhanced)](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
- [1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)

![insufficient-color-contrast-close-button](https://user-images.githubusercontent.com/26493779/110819670-b0510a00-828e-11eb-86ac-286b13d0dd63.png)

(side note: PR https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/281 unintentionally fixed contrast issues of attribution links. 😋)
